lib/tasks/one_time/2017_Q4/users.rake

```diff
@@ -0,0 +1,35 @@
+# PREREQUISITE MIGRATIONS:
+# 20171025191957_add_member_to_users
+
```

Comment:
If these is a prerequisite, then you need to check for it.  (And if the prerequisites are not satisfied, return some sort of informative error message, obv.)